src/components/LayerUI.tsx

```diff
@@ -144,7 +150,8 @@ const LayerUI = ({
         const fileHandle = await exportCanvas(
           type,
           exportedElements,
-          appState,
+          // FIXME once we split UI canvas from element canvas
+          appState as AppState,
```

Comment:
This shouldn't matter now (as it's not actually any change in code, just types), and we will be splitting the canvas hopefully soon (the PR in in progress).

An alternative is to pass a `getAppState()` getter into `LayerUI` compo.